modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -5,24 +5,27 @@ module V0
     class FacilitiesInfoController < ApplicationController
       SORT_METHODS = %w[home current alphabetical appointments].freeze
 
+      before_action :validate_sort_method_inclusion!, only: %i[schedulable]
```

Comment:
Was it possible you pulled in unintentional changes? These don't seem related at all

Reply:
You should just be able to rebase and drop the other commits.

```
git rebase origin/master -i
```

Then, change `pick` to `drop` for all of the commits that aren't yours.